src/main/scala/it/pagopa/interop/backendforfrontend/service/types/TenantProcessServiceTypes.scala

```diff
@@ -152,6 +153,13 @@ object TenantProcessServiceTypes {
       Utils.enhanceTenantAttributes(as, attributes)
   }
 
+  implicit class TenantFeatureWrapper(private val t: TenantProcess.TenantFeature) extends AnyVal {
+    def toApi: TenantFeature = t.certifier match {
+      case Some(certifier) => TenantFeature(certifier = Certifier(certifier.certifierId).some)
+      case None            => TenantFeature(certifier = None)
+    }
```

Comment:
your solution works fine, just a consideration to help future changes:
right now we only have one feature (the certifier) but we could add more of them.

In that case a `feature` would be something like
`TenantFeature(certifier: Option[Certifier], secondFeature: Option[SecondFeature])`
where only one of the fields is populated for each feature.

In this configuration, the conversion would ignore a populated `secondFeature` field, generating a `TenantFeature(None, None)` without compile warning and then a bug (unfortunately the API `TenantFeature` generated uses default parameters, which do not help).

I suggest this alternative
```suggestion
    def toApi: TenantFeature = t match {
      case TenantFeature(Some(certifier)) => TenantFeature(certifier = Certifier(certifier.certifierId).some)
      case TenantFeature(None)            => TenantFeature(certifier = None)
    }
```
this time the compiler will warn us that we are not considering the new field, preventing the bug at compile time.

(just a minor thing with a very long explanation 😂 )

Reply:
I see, thanks for the explaination, it totally makes sense!